src/toil/leader.py

```diff
@@ -335,6 +336,10 @@ def mainLoop(config, batchSystem, jobStore, rootJobWrapper):
     logger.info("Found %s jobs to start and %i jobs with successors to run",
                 len(toilState.updatedJobs), len(toilState.successorCounts))
 
+    #Set proper environment variables
+    for v in getEnv():
```

Comment:
Yep, I mixed up batch system and job store. Here's what the line not should have read:

> getEnv() should be a method of the job store class. It can't be a statically defined function in a module. Look at the other methods in AbstractJobStore and model your new method after those.